iree/turbine/kernel/compiler/host_codegen.py

```diff
@@ -60,6 +60,10 @@ def isolated_test_call(
         argument_dims = get_dynamic_dims(sig.kernel_buffer_bindings, dynamic_symbols)
         # Adding unique dynamic dims as inputs.
         input_tensors += [IndexType.get() for _ in list(dict.fromkeys(argument_dims))]
+        # Add additional dynamic symbols as inputs.
+        input_tensors += [
+            IndexType.get() for _ in set(dynamic_symbols).difference(argument_dims)
```

Comment:
Can we get this captured in the LIT test? I reckon current extend attention LIT test will fail anyways without it

Reply:
Sure, updated the lit test.